src/zenml/integrations/kubeflow/flavors/kubeflow_orchestrator_flavor.py

```diff
@@ -206,7 +200,7 @@ def is_remote(self) -> bool:
         """
         if (
             self.kubernetes_context is not None
```

Comment:
All these `is not None` checks can be removed if the context is non-optional now right?
Making it non-optional is also a breaking change, do you think we can add some code to make that less confusing for users that previously had local kubeflow stacks? E.g. if the context is `None`, we can set it to `k3d-zenml-kubeflow-{str(uuid)[:8]`?

Reply:
I added some pretty explanatory messages urging the users to set the (now mandatory) attribute, if they didn't already do that, and to use the new recipe to replace the provisioning feature that is no longer part of the client.